src/networks.ts

```diff
@@ -20,3 +20,37 @@ export const CHAIN_IDs = {
   BASE_SEPOLIA: 84532,
   ARBITRUM_SEPOLIA: 421614,
 };
+
+interface PublicNetwork {
+  name: string;
+  nativeToken: string;
+  blockExplorer: string;
+}
+
+export const PUBLIC_NETWORKS: { [chainId: number]: PublicNetwork } = {
```

Comment:
Can we split this up into TESTNETs/MAINNETs like we did with the chainIds?

Reply:
Good call - implemented: https://github.com/across-protocol/constants-v2/pull/20/commits/dedaef94e8421a4147acfcabc5c4a1fed5da0e83.